ice_setup/tests/test_paths.py

```diff
@@ -0,0 +1,38 @@
+import tempfile
+import os
+import shutil
+
+import pytest
+
+from ice_setup.ice import get_package_source, DirNotFound, VersionNotFound
+
+
+@pytest.fixture
```

Comment:
Ha - it should be, I lifted it directly from your test_distributions.py.  :)

Although I need to look into it a bit more, as I noticed on my machine that some temp dirs were not getting removed/cleaned up after tests. I need to check if this is only on test failure, though.